src/components/modal/v2/parts/views/NI/Content.jsx

```diff
@@ -0,0 +1,71 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState, useRef } from 'preact/hooks';
```

Comment:
```suggestion
import { useRef } from 'preact/hooks';
```
You should be safe to remove ```useState``` after removing ```expandedState```.